web/thesauruses/php/control_strucutres.json

```diff
@@ -0,0 +1,69 @@
+{
+  "meta": {
+    "language": "php",
+    "language_version": "7.4",
+    "language_name": "PHP"
+  },
+  "categories": {
+    "Conditional Statements (ifs)": [
+      "if_conditional",
+      "if_else_conditional",
+      "if_elseif_conditional",
+      "if_elseif_else_conditional"
+    ],
+    "Loops": [
+      "while_loop",
+      "do_while_loop",
+      "until_loop",
+      "do_until_loop",
+      "for_loops",
+      "foreach_loops"
+    ],
+    "Iterations": [
+      "map_iteration",
+      "filter_iteration"
+    ]
+  },
+  "control_structures": {
+    "if_conditional": {
+      "name": "If conditional",
+      "code": "if( $true_condition ) {\n    //statement\n}"
+    },
+    "if_else_conditional": {
+      "name": "If/Else conditional",
+      "code": "if( $condition ) {\n    // statement\n} else {\n    // else statement\n}"
+    },
+    "if_elseif_conditional": {
+      "name": "If/ElseIf conditional",
+      "code": "if( $condition ) {\n    // statement\n} elseif ( $othercondition ) {\n    // elseif statement\n}"
+    },
+    "if_elseif_else_conditional": {
+      "name": "If/ElseIf/Else conditional",
+      "code": "if( $condition ) {\n    // statement\n} elseif ( $othercondition ) {\n    // elseif statement\n} else {\n    // else statement\n}"
+    },
+    "while_loop": {
+      "name": "While loop",
+      "code": "while( $condition ) {\n    // code to be executed\n}"
+    },
+    "do_while_loop": {
+      "name": "Do/While loop",
+      "code": "do {\n    //executable code\n} while ( $condition );"
+    },
+    "for_loops": {
+      "name": "For loop",
+      "code": "for ( $intial_counter, $test_counter, $increment ) {\n    // executable code\n}"
```

Comment:
The commas here should be semicolons.

Reply:
Good catch, thanks!

Reply:
Easy notice since I find I tend to do this from time to time too! (PHP, C++, JS, whatever)